pkg/awstags/aws_tags.go

```diff
@@ -0,0 +1,47 @@
+package awstags
```

Comment:
This package here encapsulates common functionality of creating tags for AWS API which is kind of broken.
The general goal is to make it move visible and easier to add tags.